archive/governance/README.md

```diff
@@ -0,0 +1,20 @@
+# Archived Governance Contents
+
+This folder contains outdated governance contents. Thanks for the mechanisms and members actively worked ever, which leads to what TiDB is today.
+
+## TiDB Maintainers
+
+Thanks to TiDB Maintainers for maintaining the TiDB project in an early stage.
+
+| GitHub ID                               | Name       | Native Name | Company |
+| :-------------------------------------- | :--------- | :---------- | :------ |
+| [shenli](https://github.com/shenli)     | Li Shen    | 申砾        | PingCAP |
+| [jackysp](https://github.com/jackysp)   | Jack Yu    | 于帅鹏      | PingCAP |
+| [coocood](https://github.com/coocood)   | Evan Zhou  | 周昱行      | PingCAP |
+| [winkyao](https://github.com/winkyao)   | Wink Yao   | 姚维        | PingCAP |
+| [zz-jason](https://github.com/zz-jason) | Jian Zhang | 张建        | PingCAP |
```

Comment:
@shenli @jackysp @coocood @winkyao @zz-jason please confirm if this is proper English name for you.

Reply:
OK for me.